src/vm/peimage.inl

```diff
@@ -481,8 +481,19 @@ inline BOOL PEImage::HasNativeHeader()
         return GetLoadedLayout()->HasNativeHeader();
     else
     {
-        PEImageLayoutHolder pLayout(GetLayout(PEImageLayout::LAYOUT_ANY,LAYOUT_CREATEIFNEEDED));
```

Comment:
@gkhanna79  I was assuming there were similar issues with BinderHasNativeHeader() for arm32 and the catch would resolve the arm32 issues which caused you to disable loading native image in the first place.